### ISSUE TITLE
Sound Volume is stuck

### ISSUE THREAD

**Issue by Tsirhcitna** [feross/webtorrent-desktop#862]
Version 0.12.0 on windows 10 pro x64. I'm watching a film, it has finished downloading, I'm on a laptop that has gone sleeping several times. The sound is stuck on the first level above silence and if I move it it returns to this level. Happens both with scroll wheel or manually dragging it. I can see the animation of the volume dot moving and during the half-second the dot is moved the volume is correctly set higher. Clicking the mute button is the same. It worked fine before this and it will probably work again if I restart webtorrent, just wanted to report it.
![bug](https://cloud.githubusercontent.com/assets/11230770/18140086/4a124494-6fb4-11e6-9638-1cea2cf547da.gif)

**Comment by Tsirhcitna** [comment]
Additional info: on the console when I change the volume this log appear: 
```
dispatch: setVolume->
C:\Users\user\AppData\Local\WebTorrent\app-0.12.0\resources\app.asar\build\renderer\main.js:258 dispatch: setVolume ["0.1"]
```
correctly pointing what volume I've set, but there isnt't a log for when it's being returned to the initial value you can see in the gif (event is not triggered?)

Ok I've started to debug it a little since the debugging tool is so easy to use, I've put a breakpoint on mouse click, stopped the movie and pressed on the volume slider. I then followed the change to 
WebTorrent\app-0.12.0\resources\app.asar\node_modules\react\lib\ReactDOMComponent.js
where between line 739 and 742 there's this piece of code
      ```
case 'input':
        lastProps = ReactDOMInput.getHostProps(this, lastProps);
        nextProps = ReactDOMInput.getHostProps(this, nextProps);
        break;
```
where lasProps is the volume slider (className:"volume-slider float-right") with the correct volume value and nextProps is the same but with the wrong value which is processed on the next step in the same file on line `758     this._updateDOMProperties(lastProps, nextProps, transaction);`
Don't know why this element is passed twice with the second one having always the same parameters (value:0.05) resetting the first one

**Comment by mathiasvr** [comment]
Thanks for looking into this!

This seems to be because resetting `previousVolume` [here](https://github.com/feross/webtorrent-desktop/blob/v0.12.0/src/renderer/views/player.js#L58), did not mutate the global state.
However this also seems to be fixed as of v0.13.0, but I have no idea how?

Let us know if this is still an issue after updating.
